components/list/list-item.js

```diff
@@ -22,6 +22,7 @@ class ListItem extends LinkMixin(RtlMixin(LitElement)) {
 			breakpoints: { type: Array },
 			checked: { type: Boolean, reflect: true },
 			illustrationOutside: { type: Boolean, attribute: 'illustration-outside'},
+			ref: { type: String, reflect: true },
```

Comment:
I don't love this "ref" name, which to me means "reference". Really I think you're using it as a unique identifier of some kind that consumers can set to whatever internal ID makes sense to them. In MVC we'll usually call these "key", so I think I'd prefer that name. Thoughts?

Reply:
The other possibility that comes to mind is `value`, which is what native inputs use for this kind of thing.

Reply:
That works for me. That was my intent and if MVC uses `key` then I think we should use it here too.

Reply:
Value would work as well. Is there any reason to use one over the other?

Reply:
I don't have a strong preference. `key` to me is a little more in line with the use case, whereas value -- in inputs anyway -- is actually the value that the user enters into the input, or the value that gets submitted with the input in the form.